generator/generate.js

```diff
@@ -536,13 +554,29 @@ ${generateFragments(name, primitiveFields, nonPrimitiveFields)}
       .join("\n")
     output += primitiveFields.length > 0 ? "\n" : ""
     output += nonPrimitiveFields
-      .map(([field, fieldName]) => {
+      .map(([field, fieldName, fieldArgs]) => {
+        let args = ""
         const selector = `${fieldName}ModelSelector`
         let p = `  ${field}(builder`
         p += ifTS(
-          `?: string | ${selector} | ((selector: ${selector}) => ${selector})`
+          `: string | ${selector} | ((selector: ${selector}) => ${selector}) | undefined`
```

Comment:
@jesse-savary, is it necessary to drop the optional parameter? If so it expects to pass `undefined` as a parameter each time we want to get a child with only __typename and id.